pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -87,14 +87,20 @@
       - actions.each_with_index do |entry, index|
         - display = entry[:active_s] == "hide" ? "none" : "block"
         .action{style: "display: #{display}; opacity: 0"}
-          - if entry[:param1_s] == "video"
-            %img#playimage{src: "/shared/images/play-button.png", width:"40", onclick: "return showVideo_mainvideo();"}
+          - if entry[:param1_s] == "hoc_title"
```

Comment:
I've made the updates to the cdo-homepage gsheet, so staging is in an inbetween state until this PR is merged in and the two changes are combined together.